opensearchpy/connection/http_requests.py

```diff
@@ -86,8 +91,10 @@ def __init__(
         http_compress: Any = None,
         opaque_id: Any = None,
         pool_maxsize: Any = None,
+        metrics: Any = MetricsNone(),
```

Comment:
Instead of `Any` we know that `metrics` is a `Metrics` and is required. Double check that you cannot pass `None` (should produce a mypy error)